File: app/src/main/java/com/bogdanorzea/popularmovies/model/response/VideosResponse.java
package com.bogdanorzea.popularmovies.model.response;

import com.bogdanorzea.popularmovies.model.object.Video;
import com.squareup.moshi.Json;

import java.util.List;

public class VideosResponse {

    @Json(name = "id")
    public int id;
    @Json(name = "results")
    public List<Video> results = null;
}

File: app/src/main/java/com/bogdanorzea/popularmovies/model/response/VideosResponse.kt
package com.bogdanorzea.popularmovies.model.response

import com.bogdanorzea.popularmovies.model.`object`.Video
import com.squareup.moshi.Json

class VideosResponse(@Json(name = "id") var id: Int = 0,
                     @Json(name = "results") var results: List<Video>? = null)
